### ISSUE TITLE
Search for hashtags with LookForTweets

### ISSUE THREAD

**Issue by gdaniel** [xatkit-bot-platform/xatkit-twitter-platform#3]
Is there a way to look for tweets with a given hashtag with LookForTweets?

I tried `look for #models19` but the response was `No results for: #models19`, however there are some tweets available with this hashtag here: https://twitter.com/search?q=%23models19

I tried with both the *RegExp* and *DialogFlow* matchers but they both produced the same result.

**Comment by nerlichman** [comment]
After looking into the problem, I realized that looking for hashtags works "fine". The problem comes when the hashtag to be looked is the same as the name of a channel, because slack converts it to a link.

For example, in a workspace that had a channel named "bot", when you try `look for #bot` Xatkit receives `look for <#CHDN1QA6N|bots>` instead.  

The only solution I can think of would be parse the input and if it starts with `<` get the substring between `|` and `>`. What do you think @gdaniel ?

**Comment by gdaniel** [comment]
Does it work if you look for a hastag that doesn't correspond to a Slack channel? In the example I gave above there is no channel called `#models19` but the platform didn't retrieve any tweet

**Comment by gdaniel** [comment]
@nerlichman is right, in his setup (DialogFlow + `#models19`) the platform is able to retrieve the tweets. We need to check how the RegExp matcher works with the `#` character, and how to solve the issue when the hashtag matches a channel name.

**Comment by gdaniel** [comment]
@nerlichman I don't think this issue is relevant anymore, especially if we drop the dependency to Slack. Looks like a NLP issue to me, and if this needs to be solved it's probably in the xatkit-runtime project.
